hw-probe.pl

```diff
@@ -14433,22 +14433,21 @@ ()
                 $Name = "ghostbsd";
             }
             
-            my $GhostBSDPortsVer = "";
+            my $GhostBSDVer = "";
             if($Opt{"FixProbe"}) {
-                $GhostBSDPortsVer = readFile($FixProbe_Logs."/ports_ver");
```

Comment:
Please preserve backward compatibility with old probe versions under the FixProbe branch of code.

Reply:
The reason I did not keep the backward compatibility is that GhostBSD is a rolling release and if a user want to install hw-probe they have no choice to upgrade their system to the latest. We are not on a traditional release like FreeBSD or Ubuntu. 

GhostBSD will package manager will complain that the system needs an upgrade before installing.

All that said I can put the backward compatibility with no problem.

Reply:
What does that mean?

Reply:
Is that using "/ports_ver" instead of "/ghostbsd-version"?

Reply:
It should first try to use your new `ghostbsd-version` and if not found fallback to use of `ports_ver`.